src/es/components/atoms/backLink/BackLink.js

```diff
@@ -42,26 +42,27 @@ export default class BackLink extends Shadow() {
   renderCSS () {
     this.css = /* css */`
       :host {
-        font: var(--mdx-sys-font-fix-label1);
-        color: var(--mdx-sys-color-neutral-bold4);
+        color: var(--back-link-color, var(--mdx-sys-color-neutral-bold4));
       }
+
       :host .back-link {
         display: flex;
         align-items: center;
         text-decoration: none;
-        color: var(--mdx-sys-color-neutral-bold4);
+        color: inherit;
         gap: var(--a-back-link-gap, 0.2em);
+        font: var(--mdx-sys-font-fix-label1);
       }
       :host .back-link:hover {
-        color: var(--a-color);
+        color: var(--back-link-color-hover, var(--a-color));
```

Comment:
can match the css naming rule: https://github.com/mits-gossau/web-components-toolbox/blob/master/README.md?plain=1#L46 due to class naming but could be weird with namespace... --back-link-default-back-link-color-hover

Reply:
Hm aber macht das hier Sinn? Das CSS ist ja für alle namespaces oder?